addons/website/views/website_templates.xml

```diff
@@ -53,6 +53,7 @@
             'data-is-published': 'website_published' in main_object.fields_get() and main_object.website_published,
             'data-can-publish': 'can_publish' in main_object.fields_get() and main_object.can_publish,
             'data-editable-in-backend': editable_in_backend,
+            'data-object-name': main_object._description,
```

Comment:
```suggestion
            'data-main-object-name': main_object._description if main_object else None,
```
(and move it next to the `data-main-object` line above)

Maybe we should just add this to the main-object, which should become `ir.ui.view/53/View`.
That would require a bit more change in Odoo as some code would need to be adapted.

So instead of
```html
<html data-main-object="ir.ui.view(3552,)" data-object-name="View"/>
<html data-main-object="product.template(9,)" data-object-name="Product Template"/>
```
=>
```html
<html data-main-object="product.template/9/Product Template"/>
```

It feels weird to have 2 properties out of 3 in a single attribute, I'd either have all 3 in separate attributes or all 3 in a single one?
Anyway, this might just be done later if we don't want to slow down this one I guess, at least just process the initial review of this message